script.js

```diff
@@ -1,12 +1,185 @@
 //You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+let allShows = getAllShows().sort(function (a, b) {
+  if (a.name < b.name) {
+    return -1;
+  } else if (a.name > b.name) {
+    return 1;
+  } else {
+    return 0;
+  }
+});
+let allEpisodes;
+const body = document.querySelector("body");
+let selectedShowsId = allShows.map((p) => p.id);
+let defaulShowId = "82";
+
+function getFetchForEpisodes(showsId) {
```

Comment:
Excellent idea creating a function for fetching the episodes but keep the call to `makePageForEpisode` and `makeEpisodeList` separate.